templates/metrics/css/popularity.html

```diff
@@ -9,6 +9,14 @@
   {% include "metrics/css/_css_metric_nav.html" %}
 {% endblock %}
 
+{% block properties %}
+<nav class="data-panel" style="font-size: smaller;">
```

Comment:
I don't think that we need "smaller".  Does it look OK with regular-sized text?

Reply:
I made it smaller so that it looks  different than the other text elements on the same page. However, I will remove it as you have suggested.